programs/server/Server.cpp

```diff
@@ -555,6 +555,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
             global_context->setClustersConfig(config);
             global_context->setMacros(std::make_unique<Macros>(*config, "macros", log));
             global_context->setExternalAuthenticatorsConfig(*config);
+            global_context->setAuxiliaryZooKeepersConfig(config);
```

Comment:
I think we should consistent with the main "zookeeper" config logic: https://github.com/ClickHouse/ClickHouse/pull/16627/files#diff-330af1a9f5e0bc78a4831acfc2b01918e1a906d8d4d95ec22f8434dd1c0b4743R567-R568. So instead of `setAuxiliaryZooKeepersConfig` it's will be better to add `reloadAuxiliaryZooKeeperIfChanged`.